src/components/Grants/GrantCard.tsx

```diff
@@ -1,28 +1,23 @@
-import React, { useCallback } from 'react'
+import React, { useCallback, useState } from 'react'
 
-import useFormatMessage, { useIntl } from 'decentraland-gatsby/dist/hooks/useFormatMessage'
 import { navigate } from 'decentraland-gatsby/dist/plugins/intl'
 import Time from 'decentraland-gatsby/dist/utils/date/Time'
-import { Header } from 'decentraland-ui/dist/components/Header/Header'
+import TokenList from 'decentraland-gatsby/dist/utils/dom/TokenList'
 
-import {
-  GrantWithUpdateAttributes,
-  ONE_TIME_PAYMENT_TIERS,
-  ProposalGrantCategory,
-  ProposalGrantTier,
-} from '../../entities/Proposal/types'
+import { GrantWithUpdateAttributes, ProposalGrantCategory } from '../../entities/Proposal/types'
 import { CLIFF_PERIOD_IN_DAYS } from '../../entities/Proposal/utils'
 import locations from '../../modules/locations'
-import Pill, { PillColor } from '../Common/Pill'
-import ProposalUpdate from '../Proposal/Update/ProposalUpdate'
+import { PillColor } from '../Common/Pill'
 
-import CliffNotice from './CliffNotice'
-import CliffProgress from './CliffProgress'
 import './GrantCard.css'
-import VestingProgress from './VestingProgress'
+import GrantCardHeader from './GrantCardHeader'
```

Comment:
You can move this to a new folder GrantCard and store these new components inside it. WDYT?